pkgs/development/python-modules/sanic/default.nix

```diff
@@ -68,8 +68,8 @@ buildPythonPackage rec {
   inherit doCheck;
 
   preCheck = ''
-    # Some tests depends on executables on PATH
-    PATH="$out/bin:${gunicorn}/bin:$PATH"
+    # Some tests depends on sanic on PATH
+    PATH="$out/bin:$PATH"
```

Comment:
My mistake on the previous PR. Gunicorn is already on `checkInputs` so it should be already available on `PATH`.

It is just sanic itself that isn't.